code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -304,6 +304,35 @@
 	new /obj/item/card/id/prisoner/random(src)
 	new /obj/item/radio/headset(src)
 
+/obj/structure/closet/secure_closet/brig/gulag
+	name = "labor camp locker"
+	desc = "A special locker designed to store prisoner belongings, allows access when prisoners meet their point quota."
+	locked = FALSE
+	var/registered_ID_UID
+
+/obj/structure/closet/secure_closet/brig/gulag/allowed(mob/M)
+	. = ..()
+	if(.) //we were gonna let them do it anyway
+		return TRUE
```

Comment:
```suggestion
	if(..()) //we were gonna let them do it anyway
		return TRUE
```
doesn't matter what `.` is set to as all return paths in this proc have preset values